app/src/main/java/com/mobile/tandil/kotlinbaseproject/activity/MainActivity.kt

```diff
@@ -30,14 +31,21 @@ class MainActivity : AppCompatActivity(), ListenerSetParkingSpaces {
         binding.updateButton.setOnClickListener {
             viewModel.showParkingSizeFragment()
         }
+        binding.parkingStayButton.setOnClickListener {
+            viewModel.showParkingStayFragment()
+        }
     }
 
     private fun updateUI(data: ParkingData) {
         when (data.state) {
-            CheckState.SHOW_FRAGMENT -> {
+            CheckState.SHOW_SIZE_FRAGMENT -> {
                 val newFragment: ParkingSizeUpdateDialogFragment = ParkingSizeUpdateDialogFragment.newInstance(this@MainActivity)
                 newFragment.show(supportFragmentManager, ParkingSizeUpdateDialogFragment.MAIN_DIALOG_KEY)
             }
+            CheckState.SHOW_STAY_FRAGMENT -> {
```

Comment:
please for this use a representative name, you are saying SHOW_STAY_FRAGMENT when initiating an activity

Reply:
done